nfs/controller.go

```diff
@@ -243,7 +245,12 @@ func (cs *CsiNfsService) makeNfsService(ctx context.Context, namespace, name str
 
 	// Create the endpointslice
 	portName := "nfs-server"
-	var portNumber int32 = 2049
+	port, err := strconv.Atoi(os.Getenv(EnvNFSServerPort))
+	if err != nil {
+		log.Warnf("invalid port %s - err %v", os.Getenv(EnvNFSServerPort), err)
+		port = 2049 // default to 2049 if invalid port is parsed
```

Comment:
Set 2049 as a const to make it clear why it's being used (e.g. const NFS_SERVER_PORT = 2049)

Reply:
done. will push my changes soon.